service/cloudconfigv2/master_template.go

```diff
@@ -11,109 +11,16 @@ type masterExtension struct {
-		{
-			AssetContent: string(e.certs.APIServer.CA),
-			Path:         "/etc/kubernetes/ssl/apiserver-ca.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.APIServer.Crt),
-			Path:         "/etc/kubernetes/ssl/apiserver-crt.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.APIServer.Key),
-			Path:         "/etc/kubernetes/ssl/apiserver-key.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		// Calico client.
-		{
-			AssetContent: string(e.certs.CalicoClient.CA),
-			Path:         "/etc/kubernetes/ssl/calico/client-ca.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.CalicoClient.Crt),
-			Path:         "/etc/kubernetes/ssl/calico/client-crt.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.CalicoClient.Key),
-			Path:         "/etc/kubernetes/ssl/calico/client-key.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		// Etcd client.
-		{
-			AssetContent: string(e.certs.EtcdServer.CA),
-			Path:         "/etc/kubernetes/ssl/etcd/client-ca.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.EtcdServer.Crt),
-			Path:         "/etc/kubernetes/ssl/etcd/client-crt.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.EtcdServer.Key),
-			Path:         "/etc/kubernetes/ssl/etcd/client-key.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		// Etcd server.
-		{
-			AssetContent: string(e.certs.EtcdServer.CA),
-			Path:         "/etc/kubernetes/ssl/etcd/server-ca.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.EtcdServer.Crt),
-			Path:         "/etc/kubernetes/ssl/etcd/server-crt.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.EtcdServer.Key),
-			Path:         "/etc/kubernetes/ssl/etcd/server-key.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		// Service account.
-		{
-			AssetContent: string(e.certs.ServiceAccount.CA),
-			Path:         "/etc/kubernetes/ssl/service-account-ca.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.ServiceAccount.Crt),
-			Path:         "/etc/kubernetes/ssl/service-account-crt.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		{
-			AssetContent: string(e.certs.ServiceAccount.Key),
-			Path:         "/etc/kubernetes/ssl/service-account-key.pem",
-			Owner:        FileOwner,
-			Permissions:  FilePermission,
-		},
-		// set_ownership_etcd_data_dir drop-in
-		{
-			AssetContent: set_ownership_etcd_data_dir_dropin,
```

Comment:
@kopiczko in this PR it delete a dropin for etcd data. I will added back in https://github.com/giantswarm/kvm-operator/pull/320

Reply:
Good you catched that. Thanks!